test/api/unit/models/build.test.js

```diff
@@ -89,28 +88,27 @@ describe('Build model', () => {
         site: sitePromise,
         build: factory.build({ site: sitePromise }),
       }).then((promisedValues) => {
+        // eslint-disable-next-line prefer-destructuring
         site = promisedValues.site;
         expect(site.publishedAt).to.be.null;
 
         return promisedValues.build.updateJobStatus({ status: 'success' });
       }).then(() => Site.findByPk(site.id))
-      .then((model) => {
-        expect(model.publishedAt).to.be.a('date');
-        expect(new Date().getTime() - model.publishedAt.getTime()).to.be.below(500);
-        done();
-      })
-      .catch(done);
+        .then((model) => {
```

Comment:
i'm seeign a lot of indention changes? this is your editor?  this seems to not be in sync with previous lines